### DISCUSSION TITLE
Can't get system audio and external mic to work

### DISCUSSION THREAD

**Discussion by matthwho** [ExistentialAudio/BlackHole#701] [Q&A] [answered]
Hi! I'm trying to screen-record so that I can do a VoiceOver and demonstrate something in a webpage at the same time. I've downloaded the latest and reinstalled it, and I've set it up according to instructions. I'm able to screen-record either the system audio or the external mic (through a USB preamp) but not both.

If I select "Blackhole 2ch" for the microphone in the screen-record settings, I hear the system audio only. If I select "Volt 176" (my usb preamp) I hear th mic only.

My setups are screenshotted below. Any help is appreciated!

![Screen Shot 2023-01-28 at 5 39 30 PM](https://user-images.githubusercontent.com/123847827/215299731-e8cbf108-29e1-4012-a2d0-45bae1778d77.png)
![Screen Shot 2023-01-28 at 5 39 24 PM](https://user-images.githubusercontent.com/123847827/215299732-c110f5e9-dced-4f88-9071-e55d18f1160e.png)
![Screen Shot 2023-01-28 at 5 39 00 PM](https://user-images.githubusercontent.com/123847827/215299733-bdb2a237-1835-43c5-b0f7-cd6cab52ed75.png)
![Screen Shot 2023-01-28 at 5 38 55 PM](https://user-images.githubusercontent.com/123847827/215299734-c1ed04ab-514e-471b-a4a0-2eefc29081d7.png)
![Screen Shot 2023-01-28 at 5 38 41 PM](https://user-images.githubusercontent.com/123847827/215299735-6bbc5cc3-b602-4fd1-b78e-78508f5b627a.png)

**Comment by matthwho** [comment]
Follow up: I'm able to get this all to work using OBS (which I just downloaded). Mic works and system audio works via the multi-output device, so I guess this is a workaround. But it'd be nice if it worked on screen capture as well.

**Comment by devinroth** [comment, marked as answer]
What you described is a limit of screen capture.  The solution is to use some software to mix mic and BlackHole. Like you stated OBS does the trick.

**Reply by matthwho** [reply]
Thank you!

**Comment by kadavy** [comment]
I have pretty much the same set up and same problem, only I'm not trying to capture a screen recording – I'm simply trying to record my system audio while recording my microphone input. I've tried with Hindenberg, Audacity, and Quicktime. Are there limitations for these apps as well?

**Reply by devinroth** [reply]
They will capture the mic but not the system audio. If you're on Ventura try this new app I'm building called Dipper. It's in alpha but should do the trick for you. https://discord.gg/pmWKPcxhgt

**Reply by kadavy** [reply]
Thanks! I think I fixed it by setting up both a Multi-Output Device and an Aggregate Device. I have to use software that supports 4 channels though, to work with my two-channel mixer. Audacity does.
